liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -580,12 +580,12 @@ public void checkConfigFile(File configFile, File serverDir) {
         public boolean compile(File dir) {
```

Comment:
I just noticed this method is called by https://github.com/OpenLiberty/ci.common/blob/15c8fa41d6c7a444cf78353fdaf5c7348dd74591/src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java#L1082 which compiles files when a new Java source directory is added.  With the changes below, this is also now changed to a warning instead of error if there is a compile failure.  Could you see if it makes sense in that scenario (when adding a new Java source directory)?  The same question applies to the Java test sources directory.

Reply:
See https://github.com/OpenLiberty/ci.maven/pull/696/commits/263659a3038393f39498e2af02fdcb6756e36cc6 - I think it makes most sense to keep the compilation in the `compile` method as is so that the compilation failures show up as errors and are more noticeable.